tests/admin_views/admin.py

```diff
@@ -952,6 +952,21 @@ class CityAdmin(admin.ModelAdmin):
     view_on_site = True
 
 
+class CountryInlineAdmin(admin.TabularInline):
+    model = Country
+    view_on_site = True
+
+
+class RegionAdmin(admin.ModelAdmin):
+    inlines = [CountryInlineAdmin]
+
+    def get_formset_params(self, request, obj, inline, prefix):
+        return {
+            **super().get_formset_params(request, obj, inline, prefix),
+            'form_kwargs': {'initial': {'name': 'default_country_value'}},
+        }
+
+
```

Comment:
If I will do so, then `test_changed_message_uses_form_labels` in `admin_views.test_history_view.AdminHistoryViewTests` will fail.

Reply:
Hmm. I haven't looked into the exact failure - perhaps you could detail what changed. But here are my thoughts:

1. Is a change to this now failing test acceptable given the changes we've made?
2. If not, can you use another pair of existing modeladmins without causing test failures, e.g. `RestaurantInlineAdmin` and `CityAdmin`? (If not those, look for others...)
3. Otherwise maybe the issue is caused with the tests for the existing modeladmins by overriding `initial` for the `name` field. Can you pass something else in `form_kwargs` instead? (We need not get hung up on the concept of this hook being about allowing customisation of `initial` - all we want to test is that when we use this hook are custom values are passed through when creating the formset.)

I'm quite sure with a little effort all the extra models, etc. can be avoided. We like to avoid adding excess new models as it slows down the test suite and adds more code to maintain. Also it reduces the changes made by this PR to those that are necessary.

Reply:
@pope1ni Thank you for the suggestion. I have committed the desired changes.